pkg/controller/statefulset/stateful_pod_control_test.go

```diff
@@ -93,7 +93,7 @@ func TestStatefulPodControlCreatePodExists(t *testing.T) {
 	}
 	events := collectEvents(recorder.Events)
 	if eventCount := len(events); eventCount != 0 {
-		t.Errorf("Expected 0 events when Pod and PVC exist found %d", eventCount)
+		t.Errorf("Expected 0 event when Pod and PVC exist found %d", eventCount)
```

Comment:
You could update these to better fit the conventional Go style of "actual" before "expected":

https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures

For example: `"Pod and PVC exist: got %d events, want 0"`